api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -34,4 +35,12 @@
     <bean name="fhirR4" class="ca.uhn.fhir.context.FhirContext" factory-method="forR4">
         <property name="restfulClientFactory" ref="restfulClientFactoryR4"/>
     </bean>
+    <!--    base ucum-essence file packaged with fhir-ucum-->
```

Comment:
couple of things here:

1. the fhir-ucum ucum-essence.xml file is currently outdated, i'll put up a PR to update to the latest provided file from 2017

2. imo we should treat that module as the main source of truth for UCUM translation instead of duplicating the core ucum-essence file in fhir2, but open to a discussion on that approach